src/main/java/com/onaio/steps/modelViewWrapper/HouseholdViewWrapper.java

```diff
@@ -41,7 +42,7 @@ public Household getHousehold(int nameViewId, int numberViewId, int commentsView
         TextView nameView = (TextView) activity.findViewById(nameViewId);
         TextView numberView = (TextView) activity.findViewById(numberViewId);
         String phoneNumber = numberView.getText().toString();
-        String currentDate = new SimpleDateFormat(Constants.DATE_FORMAT).format(new Date());
+        String currentDate = new SimpleDateFormat(Constants.DATE_FORMAT, Locale.ENGLISH).format(new Date());
```

Comment:
Generated dates are currently only used in the STEPS dashboard. From my understanding, the dashboard is only able to 'ingest' dates that are in the English locale. Making this configurable on the Android side might not be a good idea (right now) because it might render data indigestible on the dashboard